### PR TITLE
Dev -> Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**
Update integrations flow steps 
Update managing workspaces 
Update Apimatic integration  
Remove node from dependencies  
tw-109 Datadog integration pagination issue 
Add GitHub issue templates for users creating new GitHub issues
issue-3190 Add webhook as method to automate collection runs 
Adjust cards to add Postman Space Camp on Learning Center Home page 
TW-90 Rename VictorOps to Splunk On-Call 
Add warnings on large collection deletion  
Contributing and style guide edits 
Update importing-and-exporting-data.md 
Clarify working directory for Postman web 
TW-80 Add collection recommendation feature

### CODE REVIEW

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 1]
```diff
@@ -134,45 +134,73 @@ There are also tools on GitHub to convert the following into a Postman collectio
...
 
 You will receive a confirmation once the import has completed.
 
-<img alt="Import completed" src="https://assets.postman.com/postman-docs/successful-import.jpg"/>
+![Confirm complete import from github](https://assets.postman.com/postman-docs/import-export-github-complete.jpg)
 
 You can now view your newly imported files and generated collections in Postman.
 
-<img alt="Imported data in app" src="https://assets.postman.com/postman-docs/imported-data-in-app.jpg"/>
+![Confirm complete import from github](https://assets.postman.com/postman-docs/import-export-github-complete.jpg)
```
Comment (claireriley) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:175]: This image is a duplicate of above—was this intended?

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 2]
```diff
@@ -134,45 +134,73 @@ There are also tools on GitHub to convert the following into a Postman collectio
...
+
+![Import from bitbucket auth](https://assets.postman.com/postman-docs/import-export-bitbucket-confirm-access.jpg)
+
+In Postman, select your Bitbucket workspace, repository, and branch, then **Continue**.
+
+![Import from bitbucket auth](https://assets.postman.com/postman-docs/import-export-bitbucket-select-repo.jpg)
+
+You will see a preview of the files that will be imported to your workspace. Select **Import** to confirm and complete the import.
+
+![Confirm import from bitbucket](https://assets.postman.com/postman-docs/import-export-bitbucket-import-confirm-files.jpg)
```
Comment (claireriley) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:195]: This image is missing a border

**Review by claireriley — APPROVED** [review]
LGTM! Couple small items I noticed. 

Also, question, are we not adding borders on non-Postman images? Noticed this in multiple pages here.

File: src/pages/docs/getting-started/importing-and-exporting-data.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/importing-and-exporting-data.md:175]: It shouldn't be there. I've got to do another PR to fix it.

**Pull request merged**